### ISSUE TITLE
Section 508 accessibility issue with underlying Canvas element with no name.  

### ISSUE THREAD

**Issue by rgutierrezatx** [mozilla/pdf.js#14437]
Configuration:
- Chrome Version 96.0.4664.110 (Official Build) (64-bit)
- Windows 10
- PDF.js version: linked to official demo of pdf viewer
- Is a browser extension: no

Steps to reproduce the problem:
1. Go to official mozilla pdf viewer demo at https://mozilla.github.io/pdf.js/web/viewer.html
2. run ANDI ( https://www.ssa.gov/accessibility/andi/help/install.html )

What is the expected behavior? the underlying Canvas should have an accessible name

What went wrong? see screenshot for the error.  A name should be set or at least allow us to set this name

![Untitled](https://user-images.githubusercontent.com/22502608/148827262-232008c2-08ea-4ac5-9556-7417088c9a61.png)

**Comment by Snuffleupagus** [comment]
@brendandahl Is this somewhat expected after PR #13118 (which removed the "aria-label" on the canvas-elements), or is that not entirely relevant here?

**Comment by rgutierrezatx** [comment]
@Snuffleupagus appreciate the quick response to this issue.  We are hoping for a quick turn out since this one minor issue puts the whole thing out of compliance for Section 508 accessibility.

**Comment by marco-c** [comment]
@MReschenberg is this something you could help us with? In #13118 we removed the aria-label from the canvas as it was not needed anymore (see discussion at https://github.com/mozilla/pdf.js/pull/13118#discussion_r597542880 for the reason).
Is there any way to tell that the element doesn't need an aria-label or something like that?

**Comment by marco-c** [comment]
Maybe we can set "aria-labelledby" on the canvas to point to the div that contains it?

**Comment by Snuffleupagus** [comment]
> Maybe we can set "aria-labelledby" on the canvas to point to the div that contains it?

That would likely be problematical, since I'm assuming that it'd require adding unique `id`s to the pages in the viewer; please refer to issue #11499 and PR #11503 for context.

**Comment by MReschenberg** [comment]
> @MReschenberg is this something you could help us with? In #13118 we removed the aria-label from the canvas as it was not needed anymore (see discussion at [#13118 (comment)](https://github.com/mozilla/pdf.js/pull/13118#discussion_r597542880) for the reason). Is there any way to tell that the element doesn't need an aria-label or something like that?

What's the semantic use of the `<canvas>` element inside of the region? It looks like we're using the `<region>` to define the page area, and the canvas to draw the page background. If it's purely presentational, it should have `role=presentation`, which will also make it so it no longer requires a label attribute.

**Comment by rgutierrezatx** [comment]
has the viewer at https://mozilla.github.io/pdf.js/web/viewer.html been updated with this change?  i still get the same accessibility name issue.

**Comment by marco-c** [comment]
I wonder if this isn't a bug in the ANDI tool...

**Comment by marco-c** [comment]
Yes, I think so, I even tried with an empty page with just the canvas and role presentation and ANDI is still complaining. Probably it isn't taking into account the role.

@rgutierrezatx do you have a way to report this problem to the ANDI developers?

**Comment by marco-c** [comment]
I filed https://github.com/SSAgov/ANDI/issues/138 on the ANDI GitHub repo.

**Comment by rgutierrezatx** [comment]
> I filed [SSAgov/ANDI#138](https://github.com/SSAgov/ANDI/issues/138) on the ANDI GitHub repo.

ANDI resolved the issue in their tooling and closed your ticket.  thanks!
